component-template/{{ cookiecutter.slug }}/.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Help and Support RocketChat Channel
```

Comment:
Do we need to make that configurable somehow?

Reply:
maybe make the whole file optional (like we do the component library and postprocessing filter config)

Reply:
I would like to always have the `blank_issues_enabled: false`. Removing the file, as we do it with the others, does not work here. But this is easy enough in Jinja. The question remains, shall we bother?

Reply:
We keep that for the moment. We will change that once we have the need for it.